.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.8
```

Comment:
Il faudrait tu qu'on mette la version de python qu'on utilise? Perso je suis sur python 3.12.4.
Si ça change rien, ignore this comment.

Reply:
J'ai mis 3.8 parce que dans le `Pipfile` la version requise de python pointe sur 3.8 :)